pushnotifications/src/main/java/com/pusher/pushnotifications/api/PushNotificationService.kt

```diff
@@ -54,6 +54,8 @@ data class NOKResponse(
     get() = this.toString()
 }
 
+val unknownNOKResponse = NOKResponse("Unknown Service Error", "Something went wrong")
```

Comment:
Is there anything that we could do to make the error message more descriptive?

Reply:
If we couldn't parse the json, then we don't know what went wrong. This is as descriptive as we can be.

Reply:
Should we say then in the error message that we were unable to parse the JSON?

Reply:
Unable to parse the JSON is a symptom of something going wrong, and not describing the underlying cause.

Something went wrong if this is unable to parse the error: the service was down? there was a bad proxy in the way? Some other thing we don't know about.